config/config-sil.yaml

```diff
@@ -44,6 +44,12 @@ active_modules:
   energy_manager:
     config_module:
       switch_3ph1ph_while_charging_mode: Both
+      switch_3ph1ph_max_nr_of_switches_per_session: 5
+      switch_3ph1ph_time_hyteresis_s: 20
+      switch_3ph1ph_power_hyteresis_W: 1000
```

Comment:
typos, should be hysteresis instead of hyteresis

Reply:
fixed